CONTRIBUTING.md

```diff
@@ -67,6 +67,23 @@ Most components should use the compose framework as it offers the comprehensive
    1. `<new-component>.<types | settings | platform | blah>.tsx` (Optional)
       - Optional extra files to subdivide your code however you see fit. You can also add platform specific files as you see fit.
 
+### Packaging your component (or make it into an importable module)
```

Comment:
I think there should be brief explanation why we are packing before going into how

Reply:
From shifting this around, I'm gonna cut this section and move it to the last step after "Adding a new test". It fits a bit better there and more logical that you new component needs to be packaged in order to be used in the new test you wrote.